changelogs/fragments/68482-remove-function-update-calls.yaml

```diff
@@ -0,0 +1,2 @@
+bugfixes:
+- Removed lib/ansible/modules/packaging/os/dnf.py meant to be removed once PR 19172 was merged and updated function calls.
```

Comment:
I am not sure this makes sense.

Reply:
Updated it. Hopefully everything is good now :)